pkgs/by-name/lu/luastatus/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+# Native Build Inputs
+, cmake
+, pkg-config
+, makeWrapper
+# Dependencies
+, xorg
+, yajl
+, alsa-lib
+, libpulseaudio
+, glib
+, libnl
+, udev
+, libXau
+, libXdmcp
+, pcre2
+, pcre
+, util-linux
+, libselinux
+, libsepol
+, lua5
+, docutils
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "luastatus";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "shdown";
+    repo = "luastatus";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-whO5pjUPaCwEb2GDCIPnTk39MejSQOoRRQ5kdYEQ0Pc=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    yajl
+    alsa-lib
+    libpulseaudio
+    glib
+    libnl
+    udev
+    pcre2
+    pcre
+    util-linux
+    libselinux
+    libsepol
+    lua5
+    docutils
```

Comment:
Remove the `with xorg` here

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/256033/commits/b951cae739e573385e02458354159ce2c4bac2a1